pandas/core/indexes/datetimes.py

```diff
@@ -565,29 +544,7 @@ def to_series(self, keep_tz=lib.no_default, index=None, name=None):
         if name is None:
             name = self.name
 
-        if keep_tz is not lib.no_default:
-            if keep_tz:
-                warnings.warn(
-                    "The 'keep_tz' keyword in DatetimeIndex.to_series "
-                    "is deprecated and will be removed in a future version. "
-                    "You can stop passing 'keep_tz' to silence this warning.",
-                    FutureWarning,
-                    stacklevel=find_stack_level(inspect.currentframe()),
-                )
-            else:
-                warnings.warn(
-                    "Specifying 'keep_tz=False' is deprecated and this "
-                    "option will be removed in a future release. If "
-                    "you want to remove the timezone information, you "
-                    "can do 'idx.tz_convert(None)' before calling "
-                    "'to_series'.",
-                    FutureWarning,
-                    stacklevel=find_stack_level(inspect.currentframe()),
-                )
-        else:
-            keep_tz = True
-
-        if keep_tz and self.tz is not None:
+        if self.tz is not None:
             # preserve the tz & copy
             values = self.copy(deep=True)
```

Comment:
Can we do self.copy in both cases?

Reply:
Looks like there's a copy in both if/else branches. Did you mean use `deep=True` in both cases?

Reply:
right.  so L592-L596 could just be `values = self._values.copy()` i think